src/index.ts

```diff
@@ -79,3 +73,17 @@ export const getKeypairFromEnvironment = (variableName: string) => {
   }
   return Keypair.fromSecretKey(decodedSecretKey);
 };
+
+export const addKeypairToEnvironment = (variableName: string) => {
```

Comment:
I'd expect something that adds a keypair to be `addKeypairToEnvironment(keyPair: Keypair, name: string)` - I think we might have discussed this on Slack.

Reply:
Separately, as your comment already notes, this adds a keypair to the .env file, rather than the environment itself. The function name should reflect that, ie `addKeypairToEnvFile()`

Reply:
Ah it looks like you already updated this, just didn't reply! Woo I can merge in now.